apps/api/src/campaign/campaign.service.ts

```diff
@@ -685,6 +697,84 @@ export class CampaignService {
           },
         })
       }
+
+      const percentRaised = (actualAmount / campaign.targetAmount) * 100
+
+      if (percentRaised >= 50 && percentRaised < 90)
+        // Do not await  -> send to background
+        this.sendAmountReachedNotification(campaign, 50, actualAmount).catch((e) => console.log(e))
+      if (percentRaised >= 90 && percentRaised < 100)
+        // Do not await  -> send to background
+        this.sendAmountReachedNotification(campaign, 90, actualAmount).catch((e) => console.log(e))
+      if (percentRaised >= 100)
+        // Do not await  -> send to background
+        this.sendAmountReachedNotification(campaign, 100, actualAmount).catch((e) => console.log(e))
```

Comment:
let's add a //TODO: that in future iterations the idea is for these to be configured through AdminUI

Reply:
DONE